brainglobe_atlasapi/atlas_generation/atlas_scripts/axolotl_atlas.py

```diff
@@ -0,0 +1,267 @@
+__version__ = "0"
+
+import csv
+import glob as glob
+import time
+from pathlib import Path
+
+import numpy as np
+import pooch
+import tifffile
+from rich.progress import track
+
+from brainglobe_atlasapi import utils
+
+# from skimage import io
+from brainglobe_atlasapi.atlas_generation.mesh_utils import (
+    Region,
+    create_region_mesh,
+)
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+from brainglobe_atlasapi.structure_tree_util import get_structures_tree
+
+
+def create_atlas(working_dir, resolution):
+    ATLAS_NAME = "axolotl"
+    SPECIES = "Ambystoma mexicanum"
+    ATLAS_LINK = "https://www.nature.com/articles/s41598-021-89357-3#citeas"
+    CITATION = (
+        "Lazcano, I. et al. 2021, https://doi.org/10.1038/s41598-021-89357-3"
+    )
+    ATLAS_FILE_URL = "https://zenodo.org/records/4595016"
+    ORIENTATION = "ras"
+    ROOT_ID_LEFT = 42  # FIXME
+    ROOT_ID_RIGHT = 92  # FIXME
+    ATLAS_PACKAGER = "Name, name@gmail.com"  # FIXME
```

Comment:
This should be the three of us (@saimaabdus19, @dpshelio and me) - we can give the generic BrainGlobe email hello@brainglobe.info.